libdnf5/base/log_event.cpp

```diff
@@ -153,23 +153,52 @@ std::string LogEvent::to_string(
                 } else {
                     return ret.append(utils::sformat(_("No match for group package: {}"), *spec));
                 }
+            } else if (goal_action_is_replay(action)) {
+                return ret.append(
+                    utils::sformat(_("Cannot perform {}, no match for: {}."), goal_action_to_string(action), *spec));
```

Comment:
May I ask you to mark an order of arguments `{1}`. It will allow to change order for translators.

Reply:
Added.